### PR TITLE
remove parent shutdown duration

### PR DESCRIPTION
**Pull request by ramaraochavali**
We have removed support for hot restart and parent shutdown duration is only relevant during hot restart. This PR removes parent shutdown duration.

Finally we should deprecate/remove it in ProxyConfig API

- [ ] Ambient
- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by hzxuzhonghu — APPROVED** [review]
LGTM

File: operator/pkg/util/merge_iop.go
Comment (howardjohn) [inline operator/pkg/util/merge_iop.go:186]: I think we migght still need this in case user sets it

File: pkg/kube/inject/template.go
Comment (howardjohn) [inline pkg/kube/inject/template.go:312]: I think we can only do this if we also remove it from the proto, otherwise we should keep it

**Review by howardjohn — COMMENTED** [review]
mostly lgtm

File: pkg/kube/inject/template.go
Comment (ramaraochavali) [reply pkg/kube/inject/template.go:312]: I am planning to remove it from proto after this change is merged

File: operator/pkg/util/merge_iop.go
Comment (ramaraochavali) [reply operator/pkg/util/merge_iop.go:186]: Will remove it from proto as well

Comment (ramaraochavali) [reply operator/pkg/util/merge_iop.go:186]: Should I clean that up first or any issues with cleaning up?

**Comment by ramaraochavali** [comment]
Now that api is removed, @howardjohn can you please take a look?

**Review by howardjohn — APPROVED** [review]

**Comment by zhlsunshine** [comment]
There should be no missing place, LGTM!

**Review by zhlsunshine — APPROVED** [review]
There should be no missing place, LGTM!

**Pull request merged**
